data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/sink/opensearch/OpenSearchSink.java

```diff
@@ -98,17 +103,28 @@ public OpenSearchSink(final PluginSetting pluginSetting) {
   public void initialize() throws IOException {
     LOG.info("Initializing OpenSearch sink");
     restHighLevelClient = openSearchSinkConfig.getConnectionConfiguration().createClient();
-    indexManager = indexManagerFactory.getIndexManager(indexType, restHighLevelClient, openSearchSinkConfig);
+    configuredIndexAlias = openSearchSinkConfig.getIndexConfiguration().getIndexAlias();
     final String dlqFile = openSearchSinkConfig.getRetryConfiguration().getDlqFile();
     if (dlqFile != null) {
       dlqWriter = Files.newBufferedWriter(Paths.get(dlqFile), StandardOpenOption.CREATE, StandardOpenOption.APPEND);
     }
+    if (!isDynamicIndexAlias(configuredIndexAlias)) {
```

Comment:
I'm a little concerned about leaking management of the index into this class. I think that we could handle the decisions on dynamic vs. static index management in the `IndexManagerFactory`. This would reduce the number of changes in this class.